adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -1119,6 +1119,12 @@ def _sock_exact_recv(self, bufsize: int) -> bytearray:
             rc = bytearray(bufsize)
             mv = memoryview(rc)
             recv_len = self._sock.recv_into(rc, bufsize)
```

Comment:
There is another call to `recv_into()` in the `while` cycle below (L1134). Does it need same treatment ? (if we can agree on the treatment being correct)